src/types.ts

```diff
@@ -50,14 +50,21 @@ const Policy = t.intersection([
   })
 ]);
 
+const Feature = t.type({
+  REQUIRE_OTP: t.boolean,
+  TRANSACTION_GROUPING: t.boolean
+});
+
 export const Policies = t.type({
```

Comment:
do you think it'll be cleaner if we rename the overall umbrella of `Policies` to something like
- `StageVersion`
- `EnvVersion`
- `DistVersion`

Ideally, stuff like env/distribution name and build number can come under this also haha. Cause it's meant to be version of this app haha.

What do yall think? ^^

Reply:
cause if not currently its `Policies.policies` and `Policies.features`, and the features part dont really make sense haha

Reply:
yes agree! i was quite confused with Policies having policies and features as well!

Reply:
haha yeahh~ which naming should we pick then haha

Reply:
im fine with `DistVersion` or `EnvVersion` !

Reply:
okk~ how bout `EnvVersion` then? haha

@szenius you leh HAHA